src/event/mod.rs

```diff
@@ -170,10 +175,15 @@ impl Serialize for Value {
         S: Serializer,
     {
         match &self {
+            Value::Bytes(_) => serializer.serialize_str(&self.to_string_lossy()),
             Value::Integer(i) => serializer.serialize_i64(*i),
             Value::Float(f) => serializer.serialize_f64(*f),
             Value::Boolean(b) => serializer.serialize_bool(*b),
-            _ => serializer.serialize_str(&self.to_string_lossy()),
+            Value::Timestamp(t) => {
+                serializer.serialize_str(&t.to_rfc3339_opts(SecondsFormat::AutoSi, true))
```

Comment:
Isn't this equivalent to `to_string_lossy` for timestamps? I can see the value in being explicit, but I'm curious if you have other reasons for making the change.

Reply:
Yes, it is. I changed it to `to_string_lossy`. There was no special reason, although I had thoughts about making the timestamp format configurable, but it should be out of scope of this PR.